backend/ligand_loader.py

```
@@ -0,0 +1,1010 @@
+        
+    except Exception as e:
+        print(f"   ❌ Error loading ligands from IUPHAR: {e}")
+        import traceback
+        traceback.print_exc()
+        return 0
+    finally:
+        if should_close:
+            conn.close()
+
+
+def load_ligands_from_bindingdb(target_id: int, uniprot_id: Optional[str] = None,
```

Ruff (0.14.10)

Unused function argument: `target_id`

(ARG001)